src/libs/swapAndBridge/swapAndBridgeErrorHumanizer.ts

```diff
@@ -0,0 +1,23 @@
+import EmittableError from '../../classes/EmittableError'
+import SwapAndBridgeError from '../../classes/SwapAndBridgeError'
+import SwapAndBridgeProviderApiError from '../../classes/SwapAndBridgeProviderApiError'
+
+export function getHumanReadableSwapAndBridgeError(
+  e: EmittableError | SwapAndBridgeProviderApiError | SwapAndBridgeError | Error | any
+) {
+  // These errors should be thrown as they are
+  // as they are already human-readable
+  if (
+    e instanceof EmittableError ||
+    e instanceof SwapAndBridgeProviderApiError ||
+    e instanceof SwapAndBridgeError
+  ) {
+    return e
+  }
+
+  // Last resort (fallback) error handling
+  const message = e?.message || 'no details'
+  const errorMessage = `Unexpected error happened in the Swap & Bridge flow. Try again later or contact Ambire support. Details: <${message}>`
+
+  return new Error(errorMessage)
```

Comment:
Trim the message if it exceeds x characters

Reply:
That's an excellent idea, thanks! Protection against crazy long error messages added in https://github.com/AmbireTech/ambire-common/pull/1157/commits/b573fc69ec34383e1fd6c1bb82fabecf9b5e80bc